admin-site/src/App.svelte

```diff
@@ -0,0 +1,421 @@
+							<h3>{event.label}</h3>	
+						</button>
+					{/each}
+		{/if}
+		<hr>
+		<!-- event creation -->
+		<div>
+			<label for="title">New Event Name</label> 
+			<input type="text" id="title" value={label} on:input={setTitle}>  
+		</div>
+		<button class = "btn btn-primary" on:click={createEvent}>Create New Event</button>
+</section>
+
+<!-- //occasions list populated by looping through events of "focused" event ID -->
+<div id = "occasionList">
+	<section>
+	<h1 class="text-center">Occasions</h1>
+	<hr>
+	<button type="button" class="btn btn-primary" value ="occasionList" on:click={backToEvents}>Back To Events List</button>
+	<hr>
+		{#if events.length === 0}
+			<p>No events have been added yet</p>
+			{:else}
+					{#each events as event}
+							{#if event.id == focusedEventID && event.occasions != null && event.occasions.length > 0}
+								{#each event.occasions as occasion}	
+									<button type="button" id={occasion.id} class= 'btn btn-primary btn-block' value = {occasion.id} on:click={occasionButton}>
+										<h3>{event.label} - Occasion # {occasion.id} </h3>
+											
+									</button>
+								{/each}
+							{/if}
+					{/each}
+		{/if}
+	</section>
+</div>
+
+
+
+<div id = "openEvent">
+	<div class="container-fluid">
+			<div class="row">
+				<h5> {focusedEvent.label} - {formattedStartTime}</h5>
+			</div>
+
+			<!-- <div class="row">
+				<div class="col-md-3"> -->
+					<button type="button" class="btn btn-primary" value = "openEvent" on:click={backToOccasionList}>Back To Occasion List</button>
+				<!-- </div>
+			</div> -->
+			<hr>
+
+			<div class="row">
+				<div class="col-md-12">
+					<button type="button" class="btn btn-danger btn-block" on:click={confirmEnd}>End Occasion</button>
+				</div>
+			</div>
+
+			<div class="row">
+				<div class="col-md-12">
+					<button type="button" class="btn btn-primary btn-block" value="openEvent" on:click={showQR}><u>Show QR Code</u></button>
+				</div>
+			</div>
+
+			<div class="row">
+				<div class="col-md-12">
+					<h5>Cue Details</h5>
+					{#if focusedEvent != "undefined"}
+						{#each focusedEvent.cues as cue }
+							{#if cue.cueNumber == cueState}
+								<div id = "{cue.cueNumber}">
+									<ul>Media Domain: {cue.mediaDomain}</ul>
+									<ul>Cue Number: {cue.cueNumber}</ul>
+									<ul>Cue Action: {cue.cueAction}</ul>
+								</div>
+							{/if}
+						{/each}
+					{/if}
+					
+				</div>
+			</div>
+
+			<div class="row">
+				<div class="col-md-6">
+					<button type="button" class="btn btn-info btn-block" value="previous" on:click={changeCueState}>
+						<span class="fas fa-angle-left"></span>
+						Previous
+					</button>
+				</div>
+				<div class="col-md-6">
+					<button type="button" class="btn btn-info btn-block" value="next" on:click={changeCueState}>
+						Next
+						<span class="fas fa-angle-right"></span>
+					</button>
+				</div>
+			</div> 
```

Comment:
On mobile as well, these should split center with a gap

`[< prev]       ...        [next >]`

Reply:
Change applied